### PR TITLE
refactor(poller): add initialize method populate cache

### PR DESCRIPTION
**Pull request by celiawaggoner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +49,13 @@ Segments are pre-defined groups of customers that features can be targeted to. M
...
-#### poll
+#### initialize
 
-Start polling for raw configuration data every `pollingRateSeconds`:
+Make an initial request for changes and start polling for raw configuration data
+every `pollingRateSeconds`:
 
 ```js
-poller.poll()
+poller.initialize()
```
Comment (wendychiang) [inline README.md]: does that mean client can still call poller.poll()? (imagine I call `initialize()` first, and then stop it, the next time I should call `poll()` ?)

File: lib/poller.js [hunk 2]
```diff
@@ -29,6 +29,11 @@ class Poller extends EventEmitter {
     }
   }
 
+  async initialize () {
```
Comment (silasbw) [inline lib/poller.js]: `start` (to match `stop`)

File: README.md (hunk 1 above)
Comment (celiawaggoner) [reply README.md]: they could call `poll` after `stop` since the cache will still be populated

Comment (wendychiang) [reply README.md]: should we then not delete the poll() in readme?

Comment (celiawaggoner) [reply README.md]: hm, i'd rather just keep it simple with `start` and `stop` there's no performance impact or anything from calling `start` again after `stop`

**Pull request merged**